### ISSUE TITLE
Even when running the default, I get multiple errors

### ISSUE THREAD

**Issue by jenka13all** [shangjingbo1226/AutoPhrase#80]
When I run the default docker command, and then ./auto_phrase.sh I get many errors, starting from the Tagging step:

ERROR: not a parameter file: ./lib/english-utf8.par!
(multiple lines of this)

Autophrasing step:
POS file doesn't have enough POS tags

And finally Generating Output step:
java.io.FileNotFoundException: tmp/final_quality_multi-words.txt (No such file or directory)
java.io.FileNotFoundException: tmp/final_quality_unigrams.txt (No such file or directory)
java.io.FileNotFoundException: tmp/final_quality_salient.txt (No such file or directory)

After this, in the models/DBLP directory there are only files for Autophrase.txt, language.txt, segmentation.model and token_mapping.txt.

Am I missing a step, maybe setting a tmp file somewhere?

**Comment by jenka13all** [comment]
I found out that the issue was with downloading the english.para file in tools/treetagger/pos_tag.sh - it wasn't downloading for some reason. I downloaded the file manually and set it up in the directory with the correct permissions (755) and mounted tools/treetagger/lib as a volume and no longer had the ERROR: not a parameter file: ./lib/english-utf8.par! errors. But it was still not working so I ran the docker command with ENABLE_POS_TAGGING=0.

NOW it runs through and says that everything is great, but no files are written to models, either the docker filesystem or on the mounted volume. It's as if the program just refuses to write files and that's the main issue.
